ocs_ci/utility/utils.py

```diff
@@ -5565,3 +5565,4 @@ def clean_up_pods_for_provider(
                     pod_obj = OCP(kind=constants.POD, namespace=ns, resource_name=pod)
                     cmd = f"delete pod {pod} -n {ns} --force"
                     pod_obj.exec_oc_cmd(command=cmd)
+                    continue
```

Comment:
continue in this last line is not used, it naturally goes to the next iteration

Reply:
Addressed.